model/daal4py/dffml_model_daal4py/daal4pylr.py

```diff
@@ -0,0 +1,101 @@
+import pathlib
+import statistics
+import importlib
+import hashlib
+import os
+
+from typing import AsyncIterator, Tuple, Any, Type, List
+
+from dffml import (
+    config,
+    field,
+    entrypoint,
+    SimpleModel,
+    ModelNotTrained,
+    Accuracy,
+    Feature,
+    Features,
+    Sources,
+    Record,
+)
+
+
+@config
+class DAAL4PyLRModelConfig:
+    predict: Feature = field("Label or the value to be predicted")
+    features: Features = field("Features to train on. For SLR only 1 allowed")
+    directory: pathlib.Path = field(
+        "Directory where state should be saved",
+        default=pathlib.Path("~", ".cache", "dffml", "daal4py"),
+    )
+
+
+@entrypoint("daal4pylr")
+class DAAL4PyLRModel(SimpleModel):
+    CONFIG: DAAL4PyLRModelConfig
+
+    def __init__(self, parent) -> None:
+        super().__init__(parent)
+        self.d4p = importlib.import_module("daal4py")
+        self.pd = importlib.import_module("pandas")
+        self.np = importlib.import_module("numpy")
+        self.features = self.applicable_features
+        self._features_hash = self._feature_predict_hash()
+        self.lm = self.d4p.linear_regression_training(interceptFlag=True, streaming = True)
+        self.lm_predictor = d4p.linear_regression_prediction(streaming = True)
```

Comment:
```suggestion
        self.lm_predictor = self.d4p.linear_regression_prediction(streaming = True)
```

Reply:
I think you need to look at this example: https://github.com/IntelPython/daal4py/blob/master/examples/linear_regression_streaming.py#L69

Reply:
Yes, I saw it. I was trying streaming with prediction as well... but couldn't run stuff yet so..